### ISSUE TITLE
Allow to customise connections arguments

### ISSUE THREAD

**Issue by patrick91** [strawberry-graphql/strawberry#3733]
I might not know how to do this, but I think it would be nice if we could allow to remove/change the arguments for relay connections easily. 

My current (maybe weird) usecase is that I want to return all items for a connection, so I don't need `first`, `after`, `before`, and `last`. But I can also see people not wanting `before` and `last` :)

**Comment by wwqwq2313** [comment]
Hello, we tried to solve the issue.

This is what we did:

Modify ConnectionExtension to allow customization of relay connection arguments. Add a new connection_arguments parameter to the connection decorator to specify which arguments should be included. Update the apply method to only add the specified arguments.

You can review changes in this commit: https://github.com/wwqwq2313/strawberry-graphql-strawberry-3733/commit/3a3d072fef529bc641b79249278ef242cdc9f46f.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by bellini666** [comment]
> My current (maybe weird) usecase is that I want to return all items for a connection, so I don't need first, after, before, and last. But I can also see people not wanting before and last :)

Could that be resolved by [allowing to customize max results per connection](https://github.com/strawberry-graphql/strawberry/issues/3734), and handling `None` (or maybe `-1`) as "unlimited"? Then not passing any arguments to it would achieve the same

Or do you think it is interesting to have an option to not have the arguments at all?

**Comment by patrick91** [comment]
> Or do you think it is interesting to have an option to not have the arguments at all?

yes, I think we should allow to remove them (and/or change them) 😊
